js/components/WorkingGroupHelpers.js

```diff
@@ -0,0 +1,107 @@
+export const WORKING_GROUPS = [
```

Comment:
Not necessarily for this PR, but future considerations. Do we have any way of retrieving lists of Working Groups in the Eclipse Foundation? Getting this data via AJAX would be ideal, as its one less place to update if we add a working group (like MicroProfile, or Eclipse IDE).

Reply:
That would be great if there is a data to retrieve the list of this, add to TODO @chrisguindon

Reply:
I think this is a good idea! Let's create an issue for that. This will be marked as an enhancement for the next update!